client/stripe.js

```diff
@@ -1,15 +1,24 @@
-Meteor.startup(function () {
+loadScript('https://js.stripe.com/v2/', function (err, script) {
+  if (!err) {
+    loadScript('https://checkout.stripe.com/checkout.js', function (err, script) {
+      if (!err) {
+        Meteor.startup(function () {
 
-  Stripe = window.Stripe;
-  StripeCheckout = window.StripeCheckout;
-  var config = __meteor_runtime_config__.stripe;
+          Stripe = window.Stripe;
+          StripeCheckout = window.StripeCheckout;
+          var config = __meteor_runtime_config__.stripe;
 
-  Stripe.getCheckoutHandler = function (options) {
```

Comment:
Here allow this API to be like this. But get the handler via a callback.
Trigger the callback, if both these scripts has been loaded.